azure-bundle/NOTICE

```diff
@@ -7,9 +7,9 @@ The Apache Software Foundation (http://www.apache.org/).
 
 --------------------------------------------------------------------------------
 
-NOTICE for Group: com.fasterxml.jackson.core  Name: jackson-core  Version: 2.13.5
-NOTICE for Group: com.fasterxml.jackson.core  Name: jackson-databind  Version: 2.13.5
-NOTICE for Group: com.fasterxml.jackson.dataformat  Name: jackson-dataformat-xml  Version: 2.13.5
+NOTICE for Group: com.fasterxml.jackson.core  Name: jackson-core  Version: 2.17.2
+NOTICE for Group: com.fasterxml.jackson.core  Name: jackson-databind  Version: 2.17.2
+NOTICE for Group: com.fasterxml.jackson.dataformat  Name: jackson-dataformat-xml  Version: 2.17.2
```

Comment:
Minor: it would be nice to make it clear using indentation where the copied NOTICE content begins and ends.

Reply:
Done !